helm/templates/statefulset.yaml

```diff
@@ -32,6 +32,14 @@ spec:
       - name: immudb-storage
         persistentVolumeClaim:
           claimName: {{ include "immudb.fullname" . }}
+      initContainers:
+        - command: ["/bin/chown", "-R", "3322:3322", "/data"]
```

Comment:
This uses hardcoded uid/gid. In fips we want to use distroless:noroot images which use completely different uid/gid, we should make it parametrizable I believe.

Also kubernetes shoudl automatically ensure the GID is set properly when fsGroup is set in the security context: https://kubernetes.io/docs/tasks/configure-pod-container/security-context/#configure-volume-permission-and-ownership-change-policy-for-pods - can't we use that functionality ? It would not require init container that runs as root (in some clusters its not allowed to run any container as a root).

Reply:
I agree, we can use securityContext and add the runAsUser/runAsGroup/fsGroup to set this

Reply:
I'm not sure it will do, since the directory is mounted 0755, and we will need 0775. I'm doing some tests on it

Reply:
Interesting. If using securityContext.fsGroup, directory is mounted as 2775. So that will work.

Reply:
Commit fc9d9d49 should address this; now we use securityContext.fsGroup and it work